packages/react/src/components/CardEditor/CardEditor.jsx

```diff
@@ -158,6 +158,8 @@ const defaultProps = {
     openJSONButton: 'Open JSON editor',
     searchPlaceHolderText: 'Enter a search',
     editDataItems: 'Edit data items',
+    fit: 'Contain',
```

Comment:
These defaults are being supplied in [this](https://github.com/carbon-design-system/carbon-addons-iot-react/blob/next/packages/react/src/components/CardEditor/CardEditForm/CardEditFormItems/ImageCardFormItems/ImageCardFormSettings.jsx#L95) component. If we do update these perhaps we change there?